src/shared/utils/upgradeForm.js

```diff
@@ -140,3 +143,56 @@ export function shouldRenderCancelLink(cancelAtPeriodEnd, plan, trialStatus) {
 
   return true
 }
+
+// Pro Plan Utils
+export const calculatePriceProPlan = ({ seats, baseUnitPrice }) => {
+  return Math.floor(seats) * baseUnitPrice
+}
+
+function calculateDefaultSeatsProPlan({
```

Comment:
Mainly thinking of creating very similar utils for pro, sentry, team, etc, because otherwise there's the notion on every component that you must know if you're a certain plan. That being said, this could be a place we can find reusability, hard to say.